dev-go/twofactor/Manifest

```diff
@@ -1 +1 @@
-DIST twofactor-0_p20170917.tar.gz 6637 BLAKE2B e6090a420f7d98063aa01f1c88c1dbe20103cdc4fa30d8fa9b235431f2d8cdcf1aee3cf7773a97916716aa8d0e257adc3f854db138da7c2541eef7cf1ebd493b SHA512 d3cd43d7fe2c819fa9a4ea1eacdb9508d20d54d638239c379fa8ca6ca83ecd67a06aff054c023b55e4555a227a725fd21caace55159a544fd389e30e67c363ad
+DIST twofactor-0_p20180418.tar.gz 22515 BLAKE2B 460098bf0ff570a896eb9b596ecee05ecf35219aa6ad8a5b2e8763740f588c158d186941e873623c291694eb35a2390feb372e04adf1a667aed5f884fb0b436d SHA512 ed02ab278f753d05f34b8a1e1656174cf36ce7526501d06056bdf66aaa714515a4f585b0216f987fa1cc8192b0ecfd685c729bbdb25ffe7522ba7cb8076ff026
```

Comment:
removal and addition should go in two different commits

Reply:
Resolved with b7ec54b1d888ae02834d36b99194aa2eb693368a